### ISSUE TITLE
mail not send

### ISSUE THREAD

**Issue by istatdi** [hrevert/HtUserRegistration#3]
i use this module and i have tested it on local but the mail is not sent to new user adress mail;

what shoud i see or configure ?

thanks

**Comment by ojhaujjwal** [comment]
I have not tested this module for sending mail. But, this should work.
This module uses [MtMail](https://github.com/mtymek/MtMail/) for sending mails. So, configure Mtmail too.
By default `Zend\Mail\Transport\SendMail` is the default transport. This won`t work probably in local machine.

**Comment by istatdi** [comment]
hi i use the Zend\Mail\Transport\Smtp. this is the config of the module :

``` php
'transport' => 'Zend\Mail\Transport\Smtp',
        'transport_options' => array(
            'host' => 'www.gmail.com',
            'connection_class' => 'login',
            'connection_config' => array(
                'username' => 'xxxx@gmail.com',
                'password' => 'xxx',
                'ssl' => 'tls',
            ),
        ),
```

**Comment by ojhaujjwal** [comment]
Okay, I will test it and try to solve it as soon as possible.

**Comment by istatdi** [comment]
I HAVE update the host  'host' => 'smtp.gmail.com' and i got this warning : 
steam_socket_enable_crypto() : this stream does not support ssl:crypto in ...library\zend\mail\protocol\smtp.php on line 186

**Comment by istatdi** [comment]
it work i receive the mail. thanks

**Comment by istatdi** [comment]
however this two thinks :
the link send to confirm is whitout domaine only :
http://projethtuserregistration/public/user/verify-email/8/vr2wzDF28dDIGbLh
it should be http://localhost/site/public/user/verify-email/8/vr2wzDF28dDIGbLh

the second that i got :
The requested URL could not be matched by routing.
if i want to confirm.

**Comment by ojhaujjwal** [comment]
That should be fixed in 39027798786ffd0c8099779c6e140399ae453da3

**Comment by ojhaujjwal** [comment]
And, open a new issue for other issues.
